app/models/assignment.rb

```diff
@@ -838,4 +892,106 @@ def update_assigned_tokens
       t.update_tokens(tokens_per_day_was, tokens_per_day)
     end
   end
+
+  def add_new_grouping_for_group(row, group)
+    # Create a new Grouping for this assignment and the newly
+    # crafted group
+    grouping = Grouping.new(assignment: self, group: group)
+    grouping.save
+
+    # Form groups
+    start_index_group_members = 2
+    (start_index_group_members..(row.length - 1)).each do |i|
+      student = Student.where(user_name: row[i])
+                    .first
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.